src/Components/LeftSidebar.elm

```diff
@@ -4,85 +4,129 @@ import Colors
 import Element exposing (..)
 import Element.Background as Background
 import Html.Attributes
+import Element.Events as Events
 import Material.Icons as Filled
 import Material.Icons.Types as MITypes exposing (Coloring(..), Icon)
 import Spa.Generated.Route as Route exposing (Route)
 
+type Model
+    = Normal
+    | Hover
 
-view : Element msg
-view =
-    column [ height fill, centerX, spacing 20, padding 15, alignLeft, htmlAttribute <| Html.Attributes.style "pointer-events" "all" ]
-        [ musicButton
-        , movieButton
-        , tvshowButton
-        , browserButton
-        , addonsButton
-        , thumbsupButton
-        , playlistButton
-        , settingsButton
-        , helpButton
-        ]
+update : Msg -> Model -> Model
+update msg model =
+    case msg of
+        Leave ->
+            Normal
+
+        Enter ->
+            Hover
+
+initialModel : Model
+initialModel =
+    Normal
 
+type Msg
+    = Enter
+    | Leave
 
-materialIconLink : ( Icon msg, Route ) -> Element msg
-materialIconLink ( icon, route ) =
-    el
-        [ Element.mouseOver
-            [ scale 1.1
-            , Background.color Colors.brandPrimary
-            ]
+view : Model -> Element Msg
```

Comment:
Your currently calling `view`, without a parameter (model) set. That will return a function instead.
So you end up with `LeftSidebar.Model -> Element LeftSidebar.Msg` instead of `Element msg`.

`LeftSidebar.Model` is the type it's missing, when calling `view`, look at `Frame.elm`